commands/createPool.js

```diff
@@ -0,0 +1,15 @@
+import CommandWithTrace from './commandWithTrace.class.js';
+import createPoolAction from '../actions/createPool.action.js'
+
+class CreatePool extends CommandWithTrace {
+  constructor(name) {
+    super(name);
+    this.description("Create Pool")
+      .argument('<name>', 'The name of the pool')  // Позиционный аргумент для имени
+      .argument('<description>', 'The description of the pool')  // Позиционный аргумент для описания
+      .argument('<type>', 'The type of the pool (MASTERNODE or EVONODE)')  // Позиционный аргумент для типа
+      .action(createPoolAction);
```

Comment:
Lets consider commands as a controllers in the traditional MVC application, that means on that layer we would like to execute subsequent functions, execute business logic and handle errors.

Lets have a separate class function and write that logic there (like handleCommand, or executeCommand)